Hello world Keep Alive using simplified scheduler interface (#82)
mesosphere/usi: examples/keep-alive-framework/README.md

# Keep-Alive Example Framework

Run the keep-alive example framework that:
- relies on our `core` module to handle Mesos connection, event handling, offer matching etc. as

>   opposed to the `simple-hello-world` example framework which uses only the Mesos client and has to 

Reviewer: ```suggestion
  opposed to the `core-hello-world` example framework which will start a task but fail at the first sight of troubles
```

>   implement all of the above mentioned pieces itself

Reviewer: ```suggestion
  this example will restart the task/s should they fail indefinitely similarly to how Marathon does it.
```

> - starts one `echo "Hello, world" && sleep N` task

Reviewer: ```suggestion
- starts N `echo "Hello, world" && sleep K` tasks
```

> - keeps restarting the task if it fails

Reviewer: ```suggestion
```
